platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceImpl.java

```diff
@@ -534,12 +535,13 @@ protected Application findFeature(Feature feature, Set<Application> applications
    */
   private final BundleStateSet getCurrentBundleStates(Set<Feature> features) {
     BundleStateSet bundleStateSet = new BundleStateSet();
+    BundleContext context = getContext();
 
     for (Feature curFeature : features) {
       for (BundleInfo curBundleInfo : curFeature.getBundles()) {
-        Bundle curBundle = getContext().getBundle(curBundleInfo.getLocation());
-        if (curBundle != null
-            && curBundle.adapt(BundleRevision.class).getTypes() != BundleRevision.TYPE_FRAGMENT) {
+        Bundle curBundle = context.getBundle(curBundleInfo.getLocation());
+
+        if (curBundle.adapt(BundleRevision.class).getTypes() != BundleRevision.TYPE_FRAGMENT) {
```

Comment:
❓ do we no longer need to check if `curBundle` is null?

Reply:
@shaundmorris Good catch, I'll put that check back